### ISSUE TITLE
NTR, pulmo: Architectural distortion 

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5630]
**Preferred term label:**
Architectural desorientation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Architectural distortion is characterized by abnormal displacement of bronchi, vessels, fissures, or septa caused by diffuse or localized lung disease, particularly interstitial fibrosis. This is visible in lung biopsy and CT scans in a distorted appearance and is usually associated with pulmonary fibrosis and accompanied by volume loss.

**Parent term (use hpo.jax.org/app)**
Abnormality of respiratory morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU:chrapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Architectural distortion of the lung
syn: Pulmonary architectural distortion
